packages/langium/test/parser/langium-parser-builder.test.ts

```diff
@@ -675,6 +677,57 @@ describe('ALL(*) parser', () => {
     });
 });
 
+describe('EOF', () => {
+    test('Handle EOF', async () => {
```

Comment:
Suggestion: Can you give these tests more descriptive names? While they explain what they do, test names should also give information on what they expect.